demos/node/customcontext.js

```diff
@@ -2,17 +2,15 @@
 
 /* eslint-disable no-console */
 
-const { createCanvas } = require('canvas');
 const Vex = require('../../build/vexflow-debug');
+
 const VF = Vex.Flow;
 
 // A custom Vex.Flow.RenderContext implementation.
-// This is just a stub for demonstration purposes that console.logs all method
-// calls and arguments.
+// This is just a stub for demonstration purposes that console.logs method calls and arguments.
```

Comment:
@roney does #1214 address #655?

Reply:
If I'm reading #655 correctly, I believe this was addressed by tommadams in https://github.com/0xfe/vexflow/pull/1161  He was the one who added that customcontext.js demo. (I just updated it to reflect the current APIs).

Reply:
What about #1157? Is it also addressed?

Reply:
@tommadams can you confirm if #655 and/or #1157 are already addressed.

Reply:
Regarding #655: [msdn](https://developer.mozilla.org/en-US/docs/Web/API/OffscreenCanvas/getContext#parameters) says that `OffscreenCanvas.getContext('2d')` returns a `CanvasRenderingContext2D` but [@types/offscreencanvas](https://www.npmjs.com/package/@types/offscreencanvas) defines a different type: `OffscreenCanvasRenderingContext2D`. So it looks like it might be the case that the JavaScript runtime treats offscreen contexts as equivalent to canvas contexts but TypeScript treats them as separate types.

We might need to take a dependency on `@types/offscreencanvas` and change the `CanvasContext` constructor to accept both a `CanvasRenderingContext2D` and an `OffscreenCanvasRenderingContext2D`. I can look at that over the weekend.

Regarding #1157: I still don't know what the original problem was. It looked as though the original poster was trying to pass some kind of React object to the `Renderer` constructor but they never actually fully explained what they were doing. I think we can close that one for now.